src/components/NcRichText/NcReferenceWidget.vue

```diff
@@ -72,9 +72,11 @@ export default {
 		const widgetRoot = ref()
 		const { width } = useElementSize(widgetRoot)
 
-		useIntersectionObserver(widgetRoot, () => {
-			nextTick(() => {
-				isVisible.value = widgetRoot.value?.isIntersecting ?? false
+		useIntersectionObserver(widgetRoot, (entries) => {
```

Comment:
There is always only one element expected, so we could omit forEach cycle
```suggestion
		useIntersectionObserver(widgetRoot, ([entry]) => {
```